tools/profiling/Hook.php

```diff
@@ -63,6 +63,10 @@ public static function coreRenderWidget($module, $registeredHookName, $params)
 
         $result = parent::coreRenderWidget($module, $registeredHookName, $params);
 
+        if ($registeredHookName === null) {
```

Comment:
Avoid error inside the contact form page

Reply:
Perhaps this would be best explained in a comment